config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-periodics-main.yaml

```diff
@@ -0,0 +1,24 @@
+periodics:
+- name: periodic-cluster-api-operator-test-main
+  interval: 1h
```

Comment:
Do we really need a periodic with this frequency? Are we observing the results in testgrid at all? I would expect we probably only need this every 24h if at all